### DISCUSSION TITLE
Don't use deprecated gdk-pixbuf-xlib

### DISCUSSION THREAD

**Discussion by City-busz** [linuxdeepin/developer-center#8540] [Developer Discussion | 开发者讨论]
Currently the gdkpixbuf go library depends on gdk-pixbuf-xlib:
https://github.com/linuxdeepin/go-lib/blob/dddda54ea9f073c511dfa77cca61232128a60a81/gdkpixbuf/gdk_pixbuf_utils.c#L11

gdk-pixbuf-xlib is archived by the developers and no longer maintained:
https://gitlab.gnome.org/Archive/gdk-pixbuf-xlib

Please remove this dependency.

**Comment by BLumia** [comment]
Thanks for the heads up! We'll look into this later.
